SpywareFilter/sections/tracking_servers.txt

```diff
@@ -38,6 +38,16 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||stat.tiara.tistory.com^
```

Comment:
Can be added as
```
||stat.tiara.
||track.tiara.
```
They still will be blocked by DNS and ad blockers.
![image](https://user-images.githubusercontent.com/8361299/101288985-1fd3db00-3802-11eb-9ab7-f589554d990b.png)

Reply:
OK.

Reply:
980bcee91d1d5d90498429e158d1c51164bcac81